sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/RocksDBFileManager.scala

```diff
@@ -320,6 +322,10 @@ class RocksDBFileManager(
     }
     logFilesInDir(localDir, log"Loaded checkpoint files " +
       log"for version ${MDC(LogKeys.VERSION_NUM, version)}")
+    logInfo(log"RocksDB file mapping after loading checkpoint version " +
```

Comment:
ditto about the length of log message

Reply:
Same as my other comment above. For this one, we would only log it on `load`. Which would happen less frequently in typical case. For situations where there is very frequent load, then this log would be useful because that is when we typically see issues.